document/js-api/index.bs

```diff
@@ -1294,22 +1307,22 @@ constructor steps are:
 1. Let |wasmPayload| be « ».
 1. [=list/iterate|For each=] |value| and |resultType| of |payload| and |types|, paired linearly,
     1. [=list/Append=] ? [=ToWebAssemblyValue=](|value|, |resultType|) to |wasmPayload|.
-1. Set **this**.\[[Type]] to |exceptionTag|.\[[Address]].
-1. Set **this**.\[[Payload]] to |wasmPayload|.
+1. Let (|store|, |exceptionAddr|) be [=exn_alloc=](|store|, |exceptionTag|.\[[Address]], |wasmPayload|)
+1. [=initialize an Exception object|Initialize=] **this** from |exceptionAddr|.
```

Comment:
I'm a bit confused where `**this**` is coming from, shouldn't it be allocated first? This is describing the call to `new`, so the object does not pre-exist.